tensorflow/compiler/xla/tests/slice_test.cc

```diff
@@ -193,13 +193,28 @@ class SliceR1Test : public ClientLibraryTestBase,
   }
 };
 
-XLA_TEST_P(SliceR1Test, DoIt) {
-  Run<float>(GetParam());
-  Run<double>(GetParam());
-  Run<uint32>(GetParam());
-  Run<int32>(GetParam());
-  Run<uint64>(GetParam());
-  Run<int64>(GetParam());
+XLA_TEST_P(SliceR1Test, DoIt_float) {
```

Comment:
The convention in other tests is to use the XLA names for types (F32, F64, ...).

Other than that, looks good.